src/redux/reducers/jobsReducer.js

```diff
@@ -0,0 +1,41 @@
+
+import {
+  FETCH_JOBS_SUCCESS,
+  FETCH_JOBS_FAILED,
+  IS_FETCHING
```

Comment:
@neelxie please separate the `IS_FETCHING` action for Namespaces and Jobs. Make them different.
I inquired about this to detail and i was told by @ckwagaba that each component should have its own `ACTION` no-matter how similar they look

Reply:
@Mubangizi have a full understanding of what is happening and how it is being used, and please kindly embrace re-usability, I fully understood @ckwagaba point, you didnt.

Reply:
I also wanted to do the same thing. But i was told not to @neexie

Reply:
@Mubangizi unless you have a detailed elaboration of how that affects performance other than hearsay, I know codebases that have this in use and the name is @neelxie not @neexie